web-server/v0.4/src/pages/RunComparison/index.js

```diff
@@ -295,9 +297,73 @@ class RunComparison extends React.Component {
       loadingPdf,
       exportModalVisible,
       loadingClusters,
+      iterationNo,
     } = this.state;
     const { selectedControllers, selectedResults, iterationParams } = this.props;
 
+    const children = [];
```

Comment:
This logic for generating the column data should be moved to `parse.js` similar to how we invoke `onGenerateIterationClusters` in the event of a cluster change on line 179.